health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FacilityPopup.js

```
@@ -1,37 +1,352 @@
-          />,
-        ]}
-        className={"facility-popup"}
-      />
+      {loader ? (
+        <LoaderWithGap />
+      ) : (
+        <PopUp
+          onClose={onClose}
+          heading={`${t(`MICROPLAN_ASSIGNMENT_FACILITY`)} ${details?.additionalDetails?.name}`}
+          children={[
+            <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)